spark-extension/src/main/scala/org/apache/spark/sql/auron/SparkUDTFWrapperContext.scala

```diff
@@ -58,8 +62,11 @@ case class SparkUDTFWrapperContext(serialized: ByteBuffer) extends Logging {
   }
 
   private val tc = TaskContext.get()
-  private val batchSize = AuronConf.BATCH_SIZE.intConf()
-  private val maxBatchMemorySize = AuronConf.SUGGESTED_BATCH_MEM_SIZE.intConf()
+  private val sparkAuronConfig: AuronConfiguration =
```

Comment:
Should this be a method on AuronAdaptor? All the invocations of the adaptor to get the configuration requests a `getInstance` call; we can abstract this away by adding `getSparkAuronConfig()` as a public method in the Adaptor, WDYT?